lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -125,7 +129,11 @@ def sanitize(value)
     end
 
     def merge_doc_types(types)
-      Hash[@parsed_hash['signatures'].flatten.map { |param| [ param['name'], param['type'] ] }].merge(types)
+      Hash[tag_params.select { |param| !param['types'].nil? }.map { |param| [ param['name'], param['types'].first ] }].merge(types)
+    end
+
+    def tag_params
+      @parsed_hash['docstring']['tags'].select { |tag| tag['tag_name'] == 'param' }
```

Comment:
I'm trying to test the PR and getting following error on this line

```
/usr/local/share/gems/gems/kafo_parsers-0.1.4/lib/kafo_parsers/puppet_strings_module_parser.rb:136:in `tag_params': private method `select' called for "tags":String (NoMethodError)
        from /usr/local/share/gems/gems/kafo_parsers-0.1.4/lib/kafo_parsers/puppet_strings_module_parser.rb:82:in `values'
        from /usr/local/share/gems/gems/kafo_parsers-0.1.4/lib/kafo_parsers/puppet_strings_module_parser.rb:18:in `parse'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/puppet_module.rb:71:in `parse'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/configuration.rb:89:in `block in modules'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/configuration.rb:89:in `map'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/configuration.rb:89:in `modules'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/configuration.rb:189:in `params'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/configuration.rb:199:in `preset_defaults_from_puppet'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/kafo_configure.rb:270:in `set_parameters'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/kafo_configure.rb:99:in `initialize'
        from /usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `new'
        from /usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run'
        from /usr/share/gems/gems/kafo-1.0.3/lib/kafo/kafo_configure.rb:154:in `run'
        from /usr/sbin/foreman-installer:8:in `<main>'
```

I'm running CentOS 7, nightly installer with patched kafo_parsers, I deleted parser cache and installed puppet-strings with following command

```
/opt/puppetlabs/puppet/bin/gem install puppet-strings
```

```
/opt/puppetlabs/puppet/bin/gem list puppet-strings

*** LOCAL GEMS ***

puppet-strings (0.99.0)

rpm -q puppet-agent
puppet-agent-1.7.1-1.el7.x86_64
```

any idea what I could do wrong? I notice puppetlabs-strings is already 0.4.0, it seems the gem is falling behind